aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -333,6 +333,11 @@ impl DeltaChangeSet {
         self.delta_change_set.entry(key)
     }
 
+    #[inline]
+    pub fn contains(&self, key: &StateKey) -> bool {
```

Comment:
Why not keep naming consistent - `contains_key`

Reply:
Good point, actually got rid of StateChange thing